gelfclient/client.py

```diff
@@ -44,6 +50,9 @@ def log(self, _fields_dict = {}, **fields_named):
                 message['host'] = message['source']
             else:
                 message['host'] = self.source
+        if 'level' not in message:
+            message['level'] = LogLevel.alert
```

Comment:
Ok, made the change and pushed the code. We're now using numbers so this means we don't couple this library with syslog. And later on, when using it, people have the option on passing in syslog's values if they want to.